packages/components/combobox/src/ComboboxInput.tsx

```diff
@@ -75,6 +75,10 @@ export const Input = forwardRef(
       ref: inputRef,
     })
 
+    const { selectedItems, selectedItem, multiple } = ctx
```

Comment:
I favor if we don't destructure from `ctx`, especially if it's only used once. As the code get more complex, it helps see at first glance what comes from the context vs the props.